task2.py

```diff
@@ -0,0 +1,11 @@
+salary = 5000  # Ежемесячная зарплата
+spend = 6000  # Траты за первый месяц
+months = 10  # Количество месяцев, которое планируется протянуть без долгов
+increase = 0.03  # Ежемесячный рост цен
+money_capital = 0
+
+for i in range(months):
+    money_capital += spend - salary
+    spend *= (1 + 0.03)
```

Comment:
```suggestion
    spend *= (1 + increase)
```
Аналогично задачи выше. используйте переменные, которые вы ввели в работе